packages/react-resizable-panels/src/PanelResizeHandle.ts

```diff
@@ -92,19 +92,19 @@ export default function PanelResizeHandle({
     };
 
     document.body.addEventListener("contextmenu", stopDraggingAndBlur);
-    document.body.addEventListener("mouseleave", stopDraggingAndBlur);
     document.body.addEventListener("mousemove", onMove);
     document.body.addEventListener("touchmove", onMove);
-    document.body.addEventListener("mouseup", stopDraggingAndBlur);
+    window.addEventListener("mouseup", stopDraggingAndBlur);
+    window.addEventListener("touchend", stopDraggingAndBlur);
 
     return () => {
       document.body.style.cursor = "";
 
       document.body.removeEventListener("contextmenu", stopDraggingAndBlur);
-      document.body.removeEventListener("mouseleave", stopDraggingAndBlur);
       document.body.removeEventListener("mousemove", onMove);
       document.body.removeEventListener("touchmove", onMove);
-      document.body.removeEventListener("mouseup", stopDraggingAndBlur);
+      window.removeEventListener("mouseup", stopDraggingAndBlur);
+      window.removeEventListener("touchend", stopDraggingAndBlur);
```

Comment:
Not for right now, but I wonder if you'll need to eventually derive (or accept) the correct `document` and `window` to support rendering into iframes / child windows opened via `window.open()`

Reply:
Maybe! I guess it depends on where this thing gets used. This works for now 😁